src/webgpu/shader/execution/expression/call/builtin/textureDimensions.spec.ts

```diff
@@ -300,6 +300,7 @@ Parameters:
   .beforeAllSubcases(t => {
     const info = kTextureFormatInfo[t.params.format];
     t.skipIfTextureFormatNotSupported(t.params.format);
+    t.selectDeviceForRenderableColorFormatOrSkipTestCase(t.params.format);
```

Comment:
I'm confused why this is needed? IIRC test isn't trying to render to anything except rgba32uint

Reply:
For this one multisampled texture requires the `GPUTextureUsage.RENDER_ATTACHMENT` usage, thus require color renderable. Add logic for sample > 1, refactor the ` isMultisampledTextureFormat` and `skipIfMultisampleNotSupportedForFormat` helper a bit